macro/Action+Editor.command

```diff
@@ -0,0 +1 @@
+[h, macro('dnd5e_AE2_attackEditor@Lib:DnD5e'): '']
```

Comment:
Not sure how I captured the Action Editor in my PR. Probably a kerfluffle when I extracted the macro set.